create_jenkins_job.py

```diff
@@ -172,7 +172,7 @@ def main(argv=None):
             job_name = job_name[0:-5]
         job_data['time_trigger_spec'] = '0 12 * * *'
         job_data['cmake_build_type'] = 'None'
-        job_data['ament_test_args_default'] = '--retest-until-fail 20'
+        job_data['ament_test_args_default'] = '--retest-until-fail 20 -LE lint'
```

Comment:
How did you test this locally? Currently the patch passes `-LE` to `ament` but it needs to pass it to `ctest`.